index.js

```diff
@@ -8,6 +13,97 @@ server.use(helmet());
 
 // endpoints here
 
+server.get("/", (req, res) => {
+  res.send(
+    "There are three kinds of people in this world, those who can count and those who can not."
+  );
+});
+
+server.get("/api/zoos", (req, res) => {
+  db("zoos")
+    .then(zoos => {
+      res.json(zoos);
+    })
+    .catch(err => {
+      res.status(500).json({
+        error: err,
```

Comment:
I recommend sending the err and the err.message - it often provides more insight into errors.